packages/admin-ui-docs/content/filter/code.mdx

```diff
@@ -0,0 +1,305 @@
+   */
+  onChange: () => void
+  /**
+   * State that handles the popover menu
+   */
+  menu: MenuState
+}
+```
+
+With additional props for each type
+
+```ts
+export interface FilterOption<T> {
+  id: string
+  label: string
+  /**
+   * Object that may contain anything 
+   */
+  value?: T
+}
+
+interface FilterState<T> extends GenericFilterState {
+  /**
+   * State that handles rendered options and search
+   */
+  combobox: ComboboxState<FilterOption<T>>
+  /**
+   * Option that is currently applied
+   */
+  appliedItem?: FilterOption<T>
+  /**
+   * Setter for appliedItem
+   */
+  setAppliedItem: (option: FilterOption<T>) => void
+  /**
+   * Shorthand function for fetching any key from the attached object (value) on appliedItem
+   */
+  getFromApplied: (key: string) => any
+}
+
+interface FilterMultipleState<T> extends GenericFilterState {
+  /**
+   * State that handles rendered options and search
+   */
+  combobox: ComboboxMultipleState<FilterOption<T>>
+  /**
+   * Options that are currently applied
+   */
+  appliedItems: FilterOption<T>[]
+  /**
+   * Options that are currently selected on popover, but not yet applied
+   */
+  selectedItems: FilterOption<T>[]
+  /**
+   * Setter for appliedItems
+   */
+  setAppliedItems: (items: FilterOption<T>[]) => void
+  /**
+   * Shorthand function for fetching any key from the attached object (value) on each applied option
+   */
+  getFromApplied: (key: string) => any[]
+}
+```
+
+</Section>
+
+<Section hasExplanation>
+
+## Composable components
+
+Filters are built using combined components. We have components today that allow the creation of simple filters, multiple selection filters, 
+simple filters with search and multiple selection filters with search. The components are
+
+| Component             | Description                                                                        |
+| --------------------- | ---------------------------------------------------------------------------------- |
+| FilterDisclosure      |  Component for the buttons that triggers the menu                                  |
+| FilterPopover         |  Popover component, this is always a parent component                              |
+| FilterListbox         |  Component for the option list                                                     |
+| FilterFooter          |  Component for the footer with clear and apply buttons                             |
+| FilterOptionCheckbox  |  Checkbox option component, this should be used only on multiple selection filters |
+| FilterOptionRadio     |  Radio option component, this should be used only on single selection filters      |
+| FilterSearchbox       |  Search box component that should be used only when building searchable filters    |
+| FilterGroup           |  A grouping component that display the disclosures as cohesive bar                 |
+
+
+</Section>
+
+<Section hasExplanation>
+
+## Filter
```

Comment:
You can move all those live examples to the overview tab

Reply:
They're not live yet because I haven't exported the components yet